src/Compat.jl

```diff
@@ -616,6 +616,44 @@ if VERSION < v"1.9.0-DEV.1163"
 
     _empty_stack(_...) = throw(ArgumentError("`stack` on an empty collection is not allowed"))
 end
+                                
+@static if !isdefined(Base, :Splat)
```

Comment:
```suggestion
if VERSION < v"1.9.0-DEV.513"
```
To make it easier in the future to figure out at what point this can be removed.

Reply:
Okay, but I think it's better to do it with a `@static if`.

Reply:
And why is that?